app/src/main/java/com/codepath/apps/critterfinder/activities/PetDetailsActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.codepath.apps.critterfinder.activities;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+
+import com.codepath.apps.critterfinder.R;
+import com.codepath.apps.critterfinder.fragments.PetDetailsFragment;
+import com.codepath.apps.critterfinder.models.PetModel;
+
+import butterknife.ButterKnife;
+
+public class PetDetailsActivity extends AppCompatActivity {
+
+    private static String EXTRA_PET = "com.codepath.apps.critterfinder.activities.details.pet";
+
+    /**
+     * Create an intent which will start the pet details activity
+     * @param context
+     * @param pet
+     * @return
+     */
+    public static Intent getStartIntent(Context context, PetModel pet) {
+        Intent intent= new Intent(context, PetDetailsActivity.class);
+        // TODO - once our pet model is parcelabe, work our magic
+        intent.putExtra(PetDetailsActivity.EXTRA_PET, "");
+        return intent;
+    }
```

Comment:
static function to allow consumers to get an intent that will launch this activity.